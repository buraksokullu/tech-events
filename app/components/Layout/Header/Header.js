import React from 'react';
import s from './Header.scss';

const Header = () => (
  <div className={s.headerHolder}>
    <div className={s.logoMenu}>
      <a href="/" className={s.logo}>
        Trivago
      </a>
    </div>
  </div>
);

export default Header;
