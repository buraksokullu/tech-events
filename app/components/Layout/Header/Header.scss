.headerHolder {
  height: 55px;
  display: flex;
  justify-content: space-between;
  background: #fff;
  border-bottom: 1px solid #2f353a;
}
