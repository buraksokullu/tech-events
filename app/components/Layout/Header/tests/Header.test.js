import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header';

describe('Header', () => {
  let headerProps = null;
  const selectEventTypeMock = jest.fn();
  beforeEach(() => {
    headerProps = {
      selectEventType: selectEventTypeMock
    };
  });

  it('Should render correctly', () => {
    const component = shallow(<Header {...headerProps} />);

    expect(component).toMatchSnapshot();
  });
});
