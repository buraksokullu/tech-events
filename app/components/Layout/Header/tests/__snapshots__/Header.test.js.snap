// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render correctly 1`] = `
<div>
  <div>
    <a
      href="/"
    >
      <TrivagoIcon />
    </a>
    <div>
      <div
        onClick={[Function]}
        role="presentation"
      >
        All Events
      </div>
      <div
        onClick={[Function]}
        role="presentation"
      >
        My Events
      </div>
    </div>
  </div>
</div>
`;
